### ISSUE TITLE
Flavor-specific documentation

### ISSUE THREAD

**Issue by rexmtorres** [Kotlin/dokka#345]
I have a multi-flavor Android module that has different documentation for some of the classes:

- myModule/src
  - flavor1
    - MyClass.kt
      ```
      /** This is flavor1's documentation */
      class MyClass { ... }
      ```
  - flavor2
    - MyClass.kt
      ```
      /** This is flavor2's documentation */
      class MyClass { ... }
      ```

I then configured my Dokka Gradle tasks like so:
```
    applicationVariants.all { variant ->
        task "dokka${variant.name.capitalize()}" (type: org.jetbrains.dokka.gradle.DokkaAndroidTask) {
            ...
            outputDirectory = "$buildDir/dokka/$variant.name"
            sourceDirs = files("src/${variant.flavorName}/java")
            ...
        }
    }
```

However, it seems that Dokka always uses the latest flavor that was configured (in this case `flavor2`) for all my Dokka tasks.  So `MyClass` for both `flavor1` and `flavor2` will have the same documentation:
```
      /** This is flavor2's documentation */
      class MyClass
```

**Comment by rexmtorres** [comment]
> That's totally strange

What's strange?  My request/setup or the behavior of Dokka?

**Comment by semoro** [comment]
Dokka's behaviour

**Comment by tedhenry100** [comment]
I'm seeing the same behaviour that @semoro observed. This must be a bug in Dokka.

**Comment by kamilok1965** [comment]
As of 0.10.0, dokka takes the first release flavor it encounters. Selecting the flavor to document is planned for the next release

**Comment by kamilok1965** [comment]
There's also a configuration option that you can use since version 0.10.0 to document specific flavour. It's called `androidVariant` and it's a String value with a name of your variant. Please note that it's not fully tested so it may not work and it for sure doesn't in the multiplatform mode

**Comment by kamilok1965** [comment]
Implemented in 2ff1c6e5751f6e1e94a346c92775c1fa94ea9e63
